Inspicio/Controllers/ImagesController.cs

```diff
@@ -107,29 +108,51 @@ public class DataToBody
                 return NotFound();
             }
 
+            // Jack Lloyd 
+            // Fetch the image by the id pass in '/5'
             var image = await _context.Images.SingleOrDefaultAsync(m => m.ImageID == id);
             if (image == null)
             {
                 return NotFound();
             }
 
-            DataToBody DataToBody = new DataToBody();
-            DataToBody.Image = image;
+            // Jack Lloyd 
+            // Create a new FullReview object
+            // This will be passed into the view as the model.
+            FullReview FullReview = new FullReview();
+            FullReview.Image = image;
 
+            // Jack Lloyd 
+            // Fetch all the comments 
+            // This may not be the best way to do this...
             var allcomments = _context.Comments;
-            List<Comment> comments = new List<Comment>();
+
+            // Jack Lloyd 
+            // The FullReview comments holds a tuple holding one comment and one username.
+            List<Tuple<Comment, String>> comments = new List<Tuple<Comment, String>>();
```

Comment:
Rosyln (and me) prefer var for type names, otherwise you get a lot of noise in the code - like this - but that's up to you. 

Most teams seem to like `var` when creating objects where the type is obvious (like this), and the opinions start to diverge a bit on `var` when it's the results of calling a method (e.g. `var user = await db.Users.FirstAsync()`) when type isn't explicated stated in the code.